e2e/utils/functions/senaite.ts

```diff
@@ -31,6 +31,9 @@ export class SENAITE {
     await this.page.locator('input[type=checkbox]').first().click();
     await expect(this.page.getByRole('button', { name: 'Receive' })).toBeEnabled();
     await this.page.locator('#receive_transition span:nth-child(1)').click();
+    await delay(2000)
+    await this.page.locator('th.select-column input[type=checkbox]').click();
+    await this.page.locator('#create_partitions_transition').click();
```

Comment:
@VaishSiddharth, is the dev server running SENAITE v2.5.0

Reply:
Nope. [This](https://github.com/mekomsolutions/senaite-docker/pull/16) PR should go in first